android/src/main/java/com/dooboolab.audiorecorderplayer/RNAudioRecorderPlayerModule.kt

```diff
@@ -381,5 +387,19 @@ class RNAudioRecorderPlayerModule(private val reactContext: ReactApplicationCont
     companion object {
         private var tag = "RNAudioRecorderPlayer"
         private var defaultFileName = "sound.mp4"
+        private var defaultFileExtensions = listOf(
+            "mp4", // DEFAULT = 0,
+            "3gp", // THREE_GPP,
+            "mp4", // MPEG_4,
+            "amr", // AMR_NB,
+            "amr", // AMR_WB,
+            "aac", // AAC_ADIF,
+            "aac", // AAC_ADTS,
+            "rtp", // OUTPUT_FORMAT_RTP_AVP,
+            "ts",  // MPEG_2_TSMPEG_2_TS,
+            "webm",// WEBM
+            "xxx", // UNUSED
+            "ogg", // OGG
```

Comment:
commas in comments could be identical here!

Reply:
I removed the commas for consistency. Let me know if you'd like to make any other adjustments.